bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java

```diff
@@ -292,12 +304,37 @@ public EntryLogger(ServerConfiguration conf,
                 logId = lastLogId;
             }
         }
-        this.leastUnflushedLogId = logId + 1;
+        this.recentlyCreatedEntryLogsStatus = new RecentEntryLogsStatus(logId + 1);
```

Comment:
Should be part of the entryLogManager.

Reply:
i kept the logic related to newlog creation and flush logs with in EntryLogger. So it should be here.